test/libyul/yulSyntaxTests/metadata_access_subobject.yul

```diff
@@ -0,0 +1,12 @@
+object "A" {
+  code {
```

Comment:
Bad indent.

Reply:
Do we really need to be that strict about indents in test files (as long as they are consistent inside the file)? If we always use tabs, we never test spaces...

Reply:
Inconsistent indents are very annoying when switching between files because you have to adjust editor settings for each new file separately. If we stick to the values from `.editorconfig` the editor can do that automatically which makes life easier. 

Still, I know it's not necessarily a good idea to reindent existing tests so I'm only complaining about this when I see someone add a completely new file using a random indent style for no reason :)

Reply:
Ah ok, I set me editor to auto-infer the indentation style from the file... :)

Reply:
Maybe I need to look into configuring that. I wonder if it will handle mixed indents well.

Reply:
converted spaces to tabs.